.github/workflows/deploy.yml

```diff
@@ -24,3 +24,38 @@ jobs:
           cloudflare_api_token: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           commit_sha: ${{ github.event.workflow_run.head_sha }}
           workflow_run_id: ${{ github.event.workflow_run.id }}
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.10.0
+
+      - name: Generate Claimable Permit
+        id: permit_generation
+        run: |
+          yarn
+          output=$(yarn "start:sign" | grep -o "https://[^ ]*" | sed -n '2p')
+          echo "Permit available at the address:"
+          echo $output
+          echo "CLAIMABLE_URL=$output" >> $GITHUB_ENV
+        env:
+          BENEFICIARY_ADDRESS: "0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd"
+          PAYMENT_TOKEN_ADDRESS: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
+          AMOUNT_IN_ETH: 0
+          CHAIN_ID: 100
+          FRONTEND_URL: "${{ env.DEPLOYMENT_URL }}"
+          RPC_PROVIDER_URL: "http://eth.ubq.fi/v1/mainnet"
```

Comment:
This doesn't work

Reply:
You mean that the RPC URL doesn't work? It's okay since it is not used during the build process to generate the permit, only by the invalidation, it is there to satisfy the env requirement. Please let me know a valid url to put there.

Reply:
Depends on what network. If you want to use mainnet I suggest checking the best ping from https://chainlist.org/

Reply:
I gave an arbitrary value because it is not used throughout the process. I will then use https://rpc.ankr.com/gnosis as it seems to be the fastest.